cves/CVE-2013-0848.yml

```diff
@@ -89,21 +91,23 @@ fixes_vcc_instructions: |
   The notes field is optional - place anything that clarifies things or you
   find interesting in there.
 fixes:
-- commit: 6abb9a901fca27da14d4fffbb01948288b5da3ba
-  note: ''
-- commit: a7153444df9040bf6ae103e0bbf6104b66f974cb
-  note: ''
-- commit: ''
-  note: ''
+  commit: 6abb9a901fca27da14d4fffbb01948288b5da3ba
+  note: 'check width more completely, avoid out of array accesses'
+  - commit: 6abb9a901fca27da14d4fffbb01948288b5da3ba
+    note: ''
+  - commit: a7153444df9040bf6ae103e0bbf6104b66f974cb
+    note: ''
+  - commit: ''
+    note: ''
 vccs:
-- commit: 3a085c6a37ad381a58eb30486beda44386058327
-  note: Identified by archeogit,
-- commit: 11f18fafcab80ef52391e6e29c5a1a8f6f22c4e3
-  note: Identified by SZZUnleashed
-- commit: 2ca5ca2957607cf9604cc6068a9ebc898ff37b06
-  note: Identified by archeogit,
-- commit: ''
-  note: ''
+  - commit: 3a085c6a37ad381a58eb30486beda44386058327
```

Comment:
The first commit here makes sense as a vulnerability causing commit, but I'm not sure if I understand the importance of the following two as they seem like large refactors where the code was just copied/pasted or moved.

I think adding a quick note explaining the reasoning for including each commit would be very helpful.